pkg/uwb-core/include/config/config.h

```diff
@@ -0,0 +1,370 @@
+/*
```

Comment:
Do you really need to also import these headers in the RIOT code ?

Reply:
Well either I import them or patch the code to comment them out.

Reply:
> patch the code to comment them out.

If you want to skip their content you can just set a CFLAGS to skip the inclusion of their content.

Reply:
How?

Reply:
In the package Makefile.include, add `CFLAGS += -DSYS_CONFIG_H` and the content of config.h won't be included. I'm not sure that's what you want btw ;)

Reply:
Not exactly, if there is not `config.h` it would still complain about it,  but maybe I can remove `config.h`, its more `mcu.h` that I'm refering to.